i18n/.js/src/main/scala/io/udash/i18n/bindings/TranslationBindingHelper.scala

```diff
@@ -0,0 +1,48 @@
+package io.udash.i18n.bindings
+
+import io.udash.logging.CrossLogging
+import org.scalajs.dom._
+import org.scalajs.dom.raw.Node
+
+private[bindings] trait TranslationBindingHelper extends CrossLogging {
+  import TranslationBindingHelper._
+
+  implicit def String2Node(str: String): Node =
+    document.createTextNode(str)
+
+  implicit def String2Nodes(str: String): Seq[Node] =
+    try {
```

Comment:
You can consider reusing the `raw` tag from scalatags to implement this:  `https://www.lihaoyi.com/scalatags/#Auto-escapingandunsanitizedInput`

Reply:
It was original idea to be honest but I can't do it. Ok, I can but it requires a lot of code, because `raw` way requires to build up a full frag.